tests/unit/sftp_test.py

```diff
@@ -18,11 +20,22 @@ def sftp_session(ssh_client_session):
         del sftp_sess  # noqa: WPS420
 
 
-@pytest.fixture
-def transmit_payload():
-    """Generate a binary test payload."""
-    uuid_name = uuid.uuid4()
-    return 'Hello, {name!s}'.format(name=uuid_name).encode()
+@pytest.fixture(
+    params=(32, 1024 + 1),
+    ids=('small-payload', 'large-payload'),
+)
+def transmit_payload(request: pytest.FixtureRequest):
```

Comment:
Since you're already adding typing to the args here, let's also annotate the return type.
```suggestion
def transmit_payload(request: pytest.FixtureRequest) -> bytes:
```